backend/report_submission/templates/report_submission/step-1.html

```diff
@@ -149,14 +153,50 @@
                 </fieldset>
                 <ul class="usa-button-group">
                     <li class="usa-button-group__item">
-                        <a href="javascript:void(0);" class="usa-button usa-button--outline">Cancel</a>
+                        <a class="usa-button usa-button--outline"
+                           aria-controls="modal-cancel"
+                           data-open-modal>Cancel</a>
                     </li>
                     <li class="usa-button-group__item">
                         <button class="usa-button" id="continue">Continue</button>
                     </li>
                 </ul>
             </fieldset>
         </form>
+        <div class="usa-modal flex-justify-start"
```

Comment:
This modal is used at least at three places (step-1, step-2, step-3). I would suggest to put that in a fragment instead and  and use it like  {% include 'report_submission/your_modal_fragment_name.html' %} wherever it needed to avoid code duplication.

Reply:
I had initially intended to do this, and then... didn't? I don't know. It's done now, though!